src/WebJobs.Script/Description/ScriptFunctionInvoker.cs

```diff
@@ -70,8 +70,9 @@ public static bool IsSupportedScriptType(ScriptType scriptType)
                     await ExecuteScriptAsync(bashPath, scriptHostArguments, parameters);
                     break;
                 case ScriptType.FSharp:
-                    scriptHostArguments = string.Format("/c fsi.exe \"{0}\"", _scriptFilePath);
-                    await ExecuteScriptAsync("cmd", scriptHostArguments, parameters);
+                    scriptHostArguments = string.Format("\"{0}\"", _scriptFilePath);
+                    string fsiPath = ResolveFSharpPath();
+                    await ExecuteScriptAsync(fsiPath, scriptHostArguments, parameters);
```

Comment:
Yeah, I actually already do that in this file `ResolveBashPath`. I'll do the same here.